### DISCUSSION TITLE
Restructuring of Active Achievements Groups, New Achievement Groups (WIP, Garbage, Demoted...)

### DISCUSSION THREAD

**Discussion by meleu** [RetroAchievements/RAIntegration#767] [Ideas]
_From @kdecks on April 8, 2018 3:22_

This issues is closely related to [RAWeb #105](https://github.com/RetroAchievements/RAWeb/issues/105) (Merging ~ Bonus ~ Entries into same game.

@GameDragon2k has been making plans to add some new groups. I'm adding this another other issues for both of us based on some recent discussions.

### I'm not sure if this needs to be in RASuite or RAWeb, it seems that it would require some worth from both sides.

The plan is to add new Active Achievement Groups 
![image](https://user-images.githubusercontent.com/32706333/38462202-ebbd7216-3a9f-11e8-8e69-962a93252734.png)
to reflect the needs of the community and to disentangle the various uses of unofficial.

1. **WIP (Work-In-Progress)**
   - This is a hybrid of local and online and would be the new unofficial.
   - The purpose is to create a more active shared workspace for collaborative work.
   - Achievements would be able to be pushed to Core from here (and to any other group).
   - New achievements can be created here without starting in local. Once saved they are saved to the network in the same way that code notes are.
   - Achievements here would be would be displayed from the game page [WIP progress flag (In RAWeb #57)](https://github.com/RetroAchievements/RAWeb/issues/57). This would help players see the progress of a set that's being worked on.
2. **Garbage**
   - Achievements that are in every Group except local when deleted will be moved here.
3. **Demoted**
   - When an achievement is demoted it is moved here. (This should be what unofficial 
4. **Bonus**
   -  [RAWeb #105] (https://github.com/RetroAchievements/RAWeb/issues/105) covers this more depth.
5. **Multi**
   - This group would be isolated, and treated like it's own set? Perhaps this one could be left out and into it's own set. I'm uncertain at the time of this writing.

Other
1. Change achievements points such that WIP, Garbage and Demoted cannot be earned by the player, in terms of the site, but can still be earned during the local sessions for testing.
2. Add a delete achievement to every group except local and Garbage.
3. Instead of: Local ⇒ Promote to Unofficial ⇒ Promote to Core, Achievements should be able to be moved to any Achievement Group, and still duplicated to local.
   - Including demotions inside the emulator

_Copied from original issue: RetroAchievements/RASuite#74_

**Comment by kdecks** [comment]
I'm not going to go into detail right now, but I don't think this is the best approach for  the long term of ``~bonus~``.

**Comment by Lucoshi** [comment]
Good idea!
